### PR TITLE
test(package_info_plus): run integration tests

### PR DESCRIPTION
**Pull request by blaugold**
## Description

Move disabled web specific unit tests to integration tests and start running them together with existing integration tests. This is easier than using `flutter test --platform chrome`. `--platform` does not seem to be official supported anyway.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/fluttercommunity/plus_plugins/issues). Indicate, which of these issues are resolved or fixed by this PR.*

*e.g.*
- *Fix #123*
- *Related #456*

## Checklist

- [x] I read the [Contributor Guide](https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I titled the PR using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0).
- [x] I did not modify the `CHANGELOG.md` nor the `pubspec.yaml` files.
- [ ] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate that with a `!` in the title as explained in [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0)).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by blaugold** [comment]
@vbuberen Hey :) Not sure who to ping to get a review on this PR. The motivation for this change is to be able to add the tests that were requested for https://github.com/fluttercommunity/plus_plugins/pull/1330.

**Comment by vbuberen** [comment]
You are just in time with your ping as I am doing some changes to do another release in nearest days.

File: packages/package_info_plus/package_info_plus/example/pubspec.yaml [hunk 1]
```diff
@@ -9,16 +9,19 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  http: ^0.13.5
```
[the remark this answers is not in the document]
Comment (blaugold) [reply packages/package_info_plus/package_info_plus/example/pubspec.yaml:12]: The existing tests that I moved make use of `http`.

[the remark this answers is not in the document]
Comment (vbuberen) [reply packages/package_info_plus/package_info_plus/example/pubspec.yaml:12]: Ah, right. Didn't see imports of it in the diff, but it is there in the code for web.

**Review by vbuberen — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**